src/molecules/ActionPanel/ActionPanel.tsx

```diff
@@ -16,45 +12,33 @@ const ActionPanelBody = styled.div`
   ${padding(scale(1), scale(2))};
 `;
 
-const ActionPanelLink = styled(Typography)`
+const ActionPanelFooter = styled(Typography)`
   background: ${props => props.theme.actionPanelBackground};
   border-top: 0.0416em solid ${props => props.theme.actionPanelBorder};
-  color: ${props => props.theme.link};
   display: block;
-  font-size: ${scale(1)};
-  font-weight: bold;
   padding: ${scale(1)};
-  text-align: center;
   text-decoration: none;
-
-  :hover {
-    color: ${props => props.theme.linkHover};
-  }
 ` as StyledComponentClass<
-  DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>,
+  DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
   Theme
 >;
 
-ActionPanelLink.defaultProps = { as: 'a' };
+ActionPanelFooter.defaultProps = { as: 'div' };
```

Comment:
This should be footer

Reply:
Oh, I thought <footer /> was for the document -- will fix.